src/compiler-debugging.md

```diff
@@ -45,10 +60,9 @@ stack backtrace:
   37: rustc_driver::run_compiler
 ```
 
-If you want line numbers for the stack trace, you can enable `debug = true` in
-your config.toml and rebuild the compiler (`debuginfo-level = 1` will also add
-line numbers, but `debug = true` gives full debuginfo). Then the backtrace will
-look like this:
+If you set `debug = true`, you will get line numbers for the stack trace. 
+`debuginfo-level = 1` will also add line numbers, but `debug = true` gives 
+full debug info. Then the backtrace will look like this:
```

Comment:
This is out of date.
```suggestion
If you set `debug = true`, you will get line numbers for the stack trace. Then the backtrace will look like this:
```
I also want to add
```
`debuginfo-level = 2` will also add information about local variables when using `gdb`, but is not enabled by default.
```
somewhere, but I'm not sure this is the best place.